autofish.lua

```diff
@@ -129,6 +130,16 @@ local function print_status()
     end
 end
 
+function setEnabled(val)
+    load_state()
+    enabled = val
+    persist_state()
+
+    if not enabled then return end
+    event_loop()
+end
```

Comment:
this is technically already handled by the `enabled` API and shouldn't be necessary here.

Reply:
without this, toggling enable in the gui does not do anything. should I just have it call my script through the equivalent of exec(...)?

Reply:
The appropriate interface would be `dfhack.run_command{'enable', 'autofish'}`, I think